VRSampleApp/VRSampleApp/Classes/PresentationLayer/Modules/VideoPlayerModule/ViewModel/VideoPlayerViewModel.m

```diff
@@ -25,8 +36,51 @@ - (instancetype)initWithVideoItem:(VideoItem *)videoItem
     self.QAModeEnabled = QAModeEnabled;
   }
   
+  [self removeOldNewLogFileWithName:kLogFileName];
+  
   return self;
 }
 
+#pragma mark - Public functions
+
+- (void)debugPrint:(NSString *)debugString {
+  
+  NSString *logFilePath = [self pathToLogFileWithName:kLogFileName];
+  NSString *dump = @"";
+  NSError *error;
+  
+  if ([NSFileManager.defaultManager fileExistsAtPath:logFilePath]) {
+    dump = [NSString stringWithContentsOfFile:logFilePath encoding:NSUTF8StringEncoding error:&error];
+  }
+  
+  dump = [NSString stringWithFormat:@"%@ :::::::::: %@", dump, debugString];
+  
+  // Write to the file
+  
+  [dump writeToFile:logFilePath atomically:YES encoding:NSUTF8StringEncoding error:&error];
+  
+  if (error) {
+    NSLog(@"Failed writing to log file: %@, Error: ", error.localizedDescription);
+  }
+}
+
+#pragma mark - Private functions
+
+- (NSString *)pathToLogFileWithName:(NSString *)fileName {
+  NSArray *paths = NSSearchPathForDirectoriesInDomains(NSDocumentDirectory, NSUserDomainMask, YES);
+  NSString *documentsDirectory = [paths firstObject];
+  
+  return [documentsDirectory stringByAppendingPathComponent:fileName];
+}
+
+- (void)removeOldNewLogFileWithName:(NSString *)fileName {
```

Comment:
same comment applies here for the name of the function. Check the one I added for Swift.